website/content/en/preview/tasks/managing-amis.md

```diff
@@ -77,6 +77,10 @@ For example, you could have:
   spec:
     # The latest AMI in this family will be used
     amiFamily: AL2
+    amiSelectorTerms:
```

Comment:
Can we also make this update to the [docs](https://github.com/aws/karpenter-provider-aws/blob/main/website/content/en/docs/tasks/managing-amis.md), [v1.0](https://github.com/aws/karpenter-provider-aws/blob/main/website/content/en/v1.0/tasks/managing-amis.md) and [v.1.1](https://github.com/aws/karpenter-provider-aws/blob/main/website/content/en/v1.1/tasks/managing-amis.md) folders?